### PR TITLE
remove faker

### PR DESCRIPTION
**Pull request by jum-s**

### CODE REVIEW

File: tests/api/fixtures/users.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const API = module.exports = {
     return signup({
       email,
       username: API.createUsername(),
-      password: faker.internet.password()
+      password: randomBytes(8)
```
Comment (maxlath) [inline tests/api/fixtures/users.js:36]: `randomBytes` returns a buffer object
```suggestion
      password: randomBytes(8).toString('base64')
```

File: tests/unit/models/028-shelf.js [hunk 2]
```diff
@@ -6,9 +6,9 @@ require('should')
 const someUserId = '1234567890a1234567890b1234567890'
 const { create, updateAttributes: update } = Shelf
 
-const faker = require('faker')
-const fakeName = faker.random.words(4)
-const fakeDesc = faker.random.words(15)
+const fakeText = require('./tests/api/fixtures/text')
```
Comment (maxlath) [inline tests/unit/models/028-shelf.js:9]:
```suggestion
const fakeText = require('tests/api/fixtures/text')
```

File: tests/api/fixtures/text.js [hunk 3]
```diff
@@ -1,16 +1,81 @@
...
 
 const capitalize = word => word[0].toUpperCase() + word.slice(1).toLowerCase()
 
-const randomWords = (numberOfWords = 5) => faker.random.words(numberOfWords)
+const randomWords = (numberOfWords, suffix = '') => getSample(lorem, numberOfWords).join(' ').concat(suffix)
 
-module.exports = {
-  humanName: () => faker.fake('{{name.firstName}} {{name.lastName}}'),
+const getSample = (array, numberOfWords = 5) => {
+  // for loop looks ~3x faster than _.sample
```
Comment (maxlath) [inline tests/api/fixtures/text.js:57]: [non-blocking]
not such a hot path that a slight perf boost is worth the cost of maintaining our own helper function, imo

**Review by maxlath — CHANGES_REQUESTED** [review]

File: tests/api/fixtures/users.js (hunk 1 above)
Comment (jum-s) [reply tests/api/fixtures/users.js]: done in 3c173fedd

File: tests/unit/models/028-shelf.js (hunk 2 above)
Comment (jum-s) [reply tests/unit/models/028-shelf.js]: done in 3c173fe

File: tests/api/fixtures/text.js (hunk 3 above)
Comment (jum-s) [reply tests/api/fixtures/text.js]: done in 3c173fe

**Pull request merged**